src/Contract/Transaction.purs

```diff
@@ -6,11 +6,15 @@ module Contract.Transaction
   , awaitTxConfirmedWithTimeout
   , awaitTxConfirmedWithTimeoutSlots
   , balanceAndSignTx
-  , balanceAndSignTxs
   , balanceAndSignTxE
+  , balanceAndSignTxs
   , balanceTx
-  , balanceTxWithAddress
   , balanceTxM
+  , balanceTxWithAdditionalUtxos
+  , balanceTxWithAddress
+  , balanceTxWithAddressesAndAdditionalUtxos
```

Comment:
We'll be using another interface for specifying balancer behavior (#1053), so these combinations won't be required in the end. I think it might make sense to remove them now